components/paragraph/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@govuk-react/paragraph",
+  "version": "0.2.1",
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@govuk-react/constants": "^0.2.1",
+    "@govuk-react/hoc": "^0.2.1",
+    "@storybook/addon-knobs": "^3.4.2",
```

Comment:
Storybook shouldn’t be a production dependency

Reply:
I noticed this in some other files, I'll make a note to move it from these to devDependancies also. Cheers for the review!